app/BUILD.bazel

```diff
@@ -214,16 +280,40 @@ in order to keep Gradle building.
 )
 for file in ALTERED_VIEW_MODELS]
 
+# Views
+
 # TODO(#1617): Remove genrules post-gradle
-# View files altered by genrules to be excluded in the app library.
+'''
+Source files for the views library that need to be altered by the view genrule.
+
+Place your file here if:
+ - It is a view file
+ - It imports resources (ex: `import org.oppia.app.R`)
+
+DO NOT place your file here if:
+ - It depends on any file not contained in the listeners, annotations, or resources libraries
+ - It does not import any resources
+ - It uses Kotlin synthetic imports
+'''
 ALTERED_VIEWS = [
     "src/main/java/org/oppia/app/customview/LessonThumbnailImageView.kt",
     "src/main/java/org/oppia/app/customview/SegmentedCircularProgressView.kt",
     "src/main/java/org/oppia/app/profile/ProfileInputView.kt",
     "src/main/java/org/oppia/app/utility/ClickableAreasImage.kt",
 ]
 
-# Source files and genrule targets for views library.
+'''
+Source files for the views library that DO NOT need to be altered by the view genrule.
```

Comment:
Ditto here--let's keep the 'place your file here' instructions sufficiently clear to help the author choose.

Reply:
okay!